packages/react/src/experimental/Information/Communities/Post/CommunityPost/index.tsx

```diff
@@ -54,6 +54,8 @@ export type CommunityPostProps = {
 
   onClick: (id: string) => void
 
+  canReact?: boolean
```

Comment:
what do you think about following the same pattern we have for `noVideoPreload`? like, `noReactions` or something